verifyml/model_tests/FEAT/tests/test_FeatureImportance.py

```diff
@@ -0,0 +1,79 @@
+# Test cases for the FeatureImportance FEAT test
+
+from ..FeatureImportance import FeatureImportance
+
+import inspect
+import pandas as pd
+
+# Read test case data
+test_data = pd.DataFrame(
+    {
+        "features": [
+            "income",
+            "gender_M",
+            "gender_F",
+            "married_No",
+            "amt",
+            "age",
+            "married_Yes",
+        ],
+        "value": [0.7, 0.4, 0.2, 0.1, 0.6, 0.5, 0.3],
+    }
+)
+
+
+def test_plot_defaults():
+    """Test that the default arguments of the plot() method are as expected."""
+
+    sig = inspect.signature(FeatureImportance.plot)
+
+    assert sig.parameters["show_n"].default == 10
+    assert sig.parameters["save_plots"].default == True
+
+
+def test_save_plots_true():
+    """Test that the plot is saved to the test object when .plot(save_plots=True)."""
+    # init test object
+    test_obj = imp_test = FeatureImportance(attrs=["gender", "married"], threshold=4)
```

Comment:
Extra `imp_test`

Reply:
In general it's better to use a more meaning name like `imp_test` than `test_obj`. Otherwise, every file / function would have the variable test obj which is technically correct but not very useful